doc/index.html

```diff
@@ -2260,6 +2270,302 @@
+library since converting the default library draw command output is done by
+just calling `nk_convert` but I decided to still make this library accessible
+since it can be useful.
+
+The draw list is based on a path buffering and polygon and polyline
+rendering API which allows a lot of ways to draw 2D content to screen.
+In fact it is probably more powerful than needed but allows even more crazy
+things than this library provides by default.
+### Stack
+The style modifier stack can be used to temporarily change a
+property inside `nk_style`. For example if you want a special
+red button you can temporarily push the old button color onto a stack
+draw the button with a red color and then you just pop the old color
+back from the stack:
+
+    nk_style_push_style_item(ctx, &ctx->style.button.normal, nk_style_item_color(nk_rgb(255,0,0)));
+    nk_style_push_style_item(ctx, &ctx->style.button.hover, nk_style_item_color(nk_rgb(255,0,0)));
+    nk_style_push_style_item(ctx, &ctx->style.button.active, nk_style_item_color(nk_rgb(255,0,0)));
+    nk_style_push_vec2(ctx, &cx->style.button.padding, nk_vec2(2,2));
+
+    nk_button(...);
+
+    nk_style_pop_style_item(ctx);
+    nk_style_pop_style_item(ctx);
+    nk_style_pop_style_item(ctx);
+    nk_style_pop_vec2(ctx);
+
+Nuklear has a stack for style_items, float properties, vector properties,
+flags, colors, fonts and for button_behavior. Each has it's own fixed size stack
+which can be changed at compile time.
+### Math
+ Since nuklear is supposed to work on all systems providing floating point
+ math without any dependencies I also had to implement my own math functions
+ for sqrt, sin and cos. Since the actual highly accurate implementations for
+ the standard library functions are quite complex and I do not need high
+ precision for my use cases I use approximations.
+ Sqrt
+ ----
+ For square root nuklear uses the famous fast inverse square root:
+ https://en.wikipedia.org/wiki/Fast_inverse_square_root with
+ slightly tweaked magic constant. While on today's hardware it is
+ probably not faster it is still fast and accurate enough for
+ nuklear's use cases. IMPORTANT: this requires float format IEEE 754
+ Sine/Cosine
+ -----------
+ All constants inside both function are generated Remez's minimax
+ approximations for value range 0...2*PI. The reason why I decided to
+ approximate exactly that range is that nuklear only needs sine and
+ cosine to generate circles which only requires that exact range.
+ In addition I used Remez instead of Taylor for additional precision:
+ www.lolengine.net/blog/2011/12/21/better-function-approximations.
+ The tool I used to generate constants for both sine and cosine
+ (it can actually approximate a lot more functions) can be
+ found here: www.lolengine.net/wiki/oss/lolremez
         -XXX.XXX-  X...X  -         X...X         -X....X           -           X....X"
 X...XXXXXXXXXXXXX...X -           "
```

Comment:
When at it, could you take a look at this too? I totally forgot about this during those years :cry:. Thanks!

Reply:
I don't get what you mean here. This file is generated